src/app/(home)/_components/hero.tsx

```
@@ -7,80 +7,79 @@
 import Image from 'next/image';
 import Link from 'next/link';
 import Balancer from 'react-wrap-balancer';
+import heroImage from '../../../../public/images/gradient-noise-purple-azure-light.png';
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)